lms/templates/courseware/course_about.html

```diff
@@ -218,6 +218,9 @@
 
       <div class="inner-wrapper">
         ${get_course_about_section(course, "overview")}
+        % if settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False) and course.licensable:
+          ${get_course_license_section(course)}
```

Comment:
HTML should be written into this template, not generated in the `get_course_license_section()` function.